documentation/jetty-documentation/src/main/asciidoc/programming-guide/server/websocket/server-websocket.adoc

```diff
@@ -41,3 +41,4 @@ If your application needs specific features that are not provided by the standar
 include::server-websocket-standard.adoc[]
 include::server-websocket-jetty.adoc[]
 include::server-websocket-filter.adoc[]
+include::../../websocket.adoc[]
```

Comment:
@sbordet I thought `websocket.adoc` was the common section which was supposed to be in both client and server websocket documentation.  I couldn't see it in the Server side documentation I have added this here.

Reply:
I don't think it's the right place, as it speaks about Jetty-specific annotations and classes, so perhaps it should be an `include` inside `server-websocket-jetty.adoc`.

I could not write a generic WebSocket architecture, since you have to refer to annotation and classes that are specific.

Also, including it in the server will cause duplicate links, so perhaps for the server documentation I would just add a paragraph that says something like "for the Jetty WebSocket architecture read xref:..." and point to the client docs (rather than including it again).

Reply:
I have reverted this change, looks like there is already some links to it from the Server documentation. Just seems strange that the common websocket documentation is contained within the client section.